packages/camera/camera_android_camerax/analysis_options.yaml

```diff
@@ -0,0 +1,8 @@
+# TODO(stuartmorgan): Remove this file and fix all the unawaited_futures
+# violations. See https://github.com/flutter/flutter/issues/127323
+
+include: ../../../analysis_options.yaml
```

Comment:
Unrelated to this pr. I wish we used absolute paths instead of relative paths when the absolute path has more meaning.  

I know that effective dart says to prefer relative paths but in this case it matters more that we are using the root analysis_options.yaml than the one 3 folders up. 
https://dart.dev/effective-dart/usage#prefer-relative-import-paths

Reply:
I agree that would be better in this case, but I don't think it's actually possible; there's no package at the root.

I guess we could make a local package with the shared options, and then depend on that in every single pubspec.yaml so we could use a `package:` URL, but that would be pretty ugly. And the local package reference in the pubspec would have to be... a relative path.